<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showReteta</class>
 <widget class="QWidget" name="showReteta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 183, 183);</string>
  </property>
  <widget class="QLabel" name="Titlu">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>17</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>13224523</width>
     <height>521325</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addReview">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>750</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover{
background-color: rgb(85, 85, 127);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
   </property>
   <property name="text">
    <string>Adauga review</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adauga o nota:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addNota">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>710</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="reviewTEXT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>700</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Nota">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>13</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>13224523</width>
     <height>521325</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>64</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover{
background-color: rgb(85, 85, 127);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
   </property>
   <property name="text">
    <string>Adauga la favorite...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>452</width>
     <height>586</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="ListaIngrediente">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 253, 193);
font: 11pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="Descriere">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 249, 168);
font: 700 11pt &quot;Segoe Script&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="Recenzii">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(247, 255, 176)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover{
background-color: rgb(85, 85, 127);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
   </property>
   <property name="text">
    <string>Sterge reteta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stergeFavorite">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>87</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover{
background-color: rgb(85, 85, 127);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
   </property>
   <property name="text">
    <string>Sterge din lista favorite</string>
   </property>
  </widget>
  <zorder>Nota</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>reviewTEXT</zorder>
  <zorder>Titlu</zorder>
  <zorder>addReview</zorder>
  <zorder>label</zorder>
  <zorder>addNota</zorder>
  <zorder>deleteButton</zorder>
  <zorder>stergeFavorite</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
